src/components/DpInput/DpInput.vue

```diff
@@ -14,6 +14,7 @@
       :class="prefixClass(classes)"
       :data-counter="dataCounter !== '' ? dataCounter : null"
       :data-dp-validate-error="dataDpValidateError || null"
+      :data-validate-error-fieldname="dataValidateErrorFieldName || null"
```

Comment:
I'd suggest to stick to the (verbose) naming for now, we can refactor it all at once, some day.

```suggestion
      :data-dp-validate-error-fieldname="dataDpValidateErrorFieldName || null"
```

Reply:
done here https://github.com/demos-europe/demosplan-ui/pull/359/commits/1967260380415c72a094563274e1ab1284657fe3